### PR TITLE
Initial frontend prototype

### PR DESCRIPTION
**Pull request by ConnorCimowsky**

### CODE REVIEW

File: src-frontend/.ember-cli [hunk 1]
```diff
@@ -5,5 +5,7 @@
 
     Setting `disableAnalytics` to true will prevent any data from being sent.
   */
-  "disableAnalytics": false
+  "disableAnalytics": false,
```
Comment (Kapin) [inline src-frontend/.ember-cli]: disable?

Comment (ConnorCimowsky) [inline src-frontend/.ember-cli]: k

**Comment by Kapin** [comment]
I think it's ok?

File: src-frontend/Brocfile.js [hunk 2]
```diff
@@ -2,7 +2,15 @@
 
 var EmberApp = require('ember-cli/lib/broccoli/ember-app');
 
-var app = new EmberApp();
+var app = new EmberApp({
```
Comment (Trinovantes) [inline src-frontend/Brocfile.js]: should also add these two options too (from http://guides.emberjs.com/v1.10.0/understanding-ember/debugging/)
```
  // Basic logging, e.g. "Transitioned into 'post'"
  LOG_TRANSITIONS: true, 

  // Extremely detailed logging, highlighting every internal
  // step made while transitioning into a route, including
  // `beforeModel`, `model`, and `afterModel` hooks, and
  // information about redirects and aborted transitions
  LOG_TRANSITIONS_INTERNAL: true
```

File: src-frontend/app/adapters/application.js [hunk 3]
```diff
@@ -0,0 +1,5 @@
+import DS from 'ember-data';
+
+export default DS.RESTAdapter.extend({
+  host: 'https://gentle-cove-4339.herokuapp.com'
```
Comment (Trinovantes) [inline src-frontend/app/adapters/application.js]: can we define all of these constants in an external file and then just do `import constants`?

File: src-frontend/app/routes/procedure.js [hunk 4]
```diff
@@ -0,0 +1,7 @@
+import Ember from 'ember';
+
+export default Ember.Route.extend({
+  model: function() {
+    return this.store.find('procedure', 1);
```
Comment (Trinovantes) [inline src-frontend/app/routes/procedure.js]: add a TODO comment to updated this later

File: src-frontend/public/assets/fixture.json [hunk 5]
```diff
@@ -0,0 +1,108 @@
+{
```
Comment (Trinovantes) [inline src-frontend/public/assets/fixture.json]: is this fixture needed anymore now that we have a working backend?

**Comment by Trinovantes** [comment]
I still prefer if we use 4 spaces for tabs in all of our source code

File: src-frontend/app/adapters/application.js (hunk 3 above)
Comment (ConnorCimowsky) [inline src-frontend/app/adapters/application.js]: Will open another PR

File: src-frontend/Brocfile.js (hunk 2 above)
Comment (ConnorCimowsky) [inline src-frontend/Brocfile.js]: Done

File: src-frontend/app/routes/procedure.js (hunk 4 above)
Comment (ConnorCimowsky) [inline src-frontend/app/routes/procedure.js]: Done

File: src-frontend/public/assets/fixture.json (hunk 5 above)
Comment (ConnorCimowsky) [inline src-frontend/public/assets/fixture.json]: Nope, good catch.

**Pull request merged**